ext/MixedModelsPRIMAExt.jl

```diff
@@ -1,27 +1,41 @@
 module MixedModelsPRIMAExt
 
-using MixedModels: MixedModels, LinearMixedModel, objective!
-using MixedModels: ProgressMeter, ProgressUnknown
+using MixedModels
+using MixedModels: Statistics
+using MixedModels: ProgressMeter, ProgressUnknown, objective!, _objective!
+using LinearAlgebra: PosDefException
 using PRIMA: PRIMA
 
+function __init__()
+    push!(MixedModels.OPTIMIZATION_BACKENDS, :prima)
+    return nothing
+end
```

Comment:
@dmbates I had to wrap this in an `__init__` so that it happens at load time and not precompile time.

Reply:
Makes sense.